CtfTraceEventSource fails to parse on arm64 Ubuntu 20.04

Seen in `ubuntu.2004.arm64.perf` running `Linux-5.15.0-85-generic-aarch64-with-glibc2.29`.

Relevant part of log (workitem `8afa85d5-d1ca-4269-a439-d9fd9082af2b`).
```text
[2023/09/20 05:24:42][INFO] ----------------------------------------------
[2023/09/20 05:24:42][INFO] Initializing logger 2023-09-20 05:24:42.917045
[2023/09/20 05:24:42][INFO] ----------------------------------------------
[2023/09/20 05:24:42][INFO] $ /home/helixbot/work/B43C0999/p/startup/Startup --app-exe pub/staticconsoletemplate --metric-type TimeToMain2 --trace-name "Startup - Static Console Template - FDD Publish_startup" --gui-app false --process-will-exit true --iterations 5 --timeout 50 --warmup true --working-dir /home/helixbot/work/B43C0999/w/B77D09DF/e --report-json-path traces/perf-lab-report.json --trace-directory traces --scenario-name "Startup - Static Console Template - FDD Publish" --environment-variables COMPlus_EnableEventLog=1
[2023/09/20 05:24:43][INFO] Running pub/staticconsoletemplate (args: "")
[2023/09/20 05:24:43][INFO] Working Directory: /home/helixbot/work/B43C0999/w/B77D09DF/e
[2023/09/20 05:24:43][INFO] Iteration set up:  (args: )
[2023/09/20 05:24:43][INFO] Iteration clean up:  (args: )
[2023/09/20 05:24:43][INFO] =============== Warm up ================
[2023/09/20 05:24:43][INFO] ***Test***
[2023/09/20 05:24:43][INFO] Added environment variable: COMPlus_EnableEventLog=1
[2023/09/20 05:24:43][INFO] Hello World!
[2023/09/20 05:24:43][INFO] 
[2023/09/20 05:24:43][INFO] 
[2023/09/20 05:24:48][INFO] Installing perf_event packages.
[2023/09/20 05:24:48][INFO] Reading package lists...
[2023/09/20 05:24:48][INFO] Building dependency tree...
[2023/09/20 05:24:48][INFO] Reading state information...
[2023/09/20 05:24:48][INFO] zip is already the newest version (3.0-11build1).
[2023/09/20 05:24:48][INFO] software-properties-common is already the newest version (0.99.9.12).
[2023/09/20 05:24:48][INFO] linux-tools-common is already the newest version (5.4.0-164.181).
[2023/09/20 05:24:48][INFO] The following packages were automatically installed and are no longer required:
[2023/09/20 05:24:48][INFO]   apport-symptoms gir1.2-goa-1.0 libasan6 libclang-common-10-dev libclang1-10
[2023/09/20 05:24:48][INFO]   libfwupdplugin1 libgcc-10-dev libomp-10-dev libomp5-10 libpython2-stdlib
[2023/09/20 05:24:48][INFO]   libpython2.7-minimal libpython2.7-stdlib libxmlb1
[2023/09/20 05:24:48][INFO]   linux-headers-5.13.0-35-generic linux-headers-5.15.0-43-generic
[2023/09/20 05:24:48][INFO]   linux-headers-5.15.0-53-generic linux-headers-5.15.0-54-generic
[2023/09/20 05:24:48][INFO]   linux-headers-5.15.0-57-generic linux-headers-5.15.0-59-generic
[2023/09/20 05:24:48][INFO]   linux-headers-5.15.0-66-generic linux-headers-5.15.0-72-generic
[2023/09/20 05:24:48][INFO]   linux-headers-5.15.0-76-generic linux-hwe-5.13-headers-5.13.0-35
[2023/09/20 05:24:48][INFO]   linux-hwe-5.15-headers-5.15.0-43 linux-hwe-5.15-headers-5.15.0-53
[2023/09/20 05:24:48][INFO]   linux-hwe-5.15-headers-5.15.0-54 linux-hwe-5.15-headers-5.15.0-57
[2023/09/20 05:24:48][INFO]   linux-hwe-5.15-headers-5.15.0-59 linux-hwe-5.15-headers-5.15.0-66
[2023/09/20 05:24:48][INFO]   linux-hwe-5.15-headers-5.15.0-72 linux-hwe-5.15-headers-5.15.0-76
[2023/09/20 05:24:48][INFO]   linux-image-5.13.0-35-generic linux-image-5.13.0-37-generic
[2023/09/20 05:24:48][INFO]   linux-image-5.15.0-43-generic linux-image-5.15.0-53-generic
[2023/09/20 05:24:48][INFO]   linux-image-5.15.0-54-generic linux-image-5.15.0-57-generic
[2023/09/20 05:24:48][INFO]   linux-image-5.15.0-59-generic linux-image-5.15.0-66-generic
[2023/09/20 05:24:48][INFO]   linux-image-5.15.0-72-generic linux-image-5.15.0-76-generic
[2023/09/20 05:24:48][INFO]   linux-modules-5.13.0-35-generic linux-modules-5.13.0-37-generic
[2023/09/20 05:24:48][INFO]   linux-modules-5.15.0-43-generic linux-modules-5.15.0-53-generic
[2023/09/20 05:24:48][INFO]   linux-modules-5.15.0-54-generic linux-modules-5.15.0-57-generic
[2023/09/20 05:24:48][INFO]   linux-modules-5.15.0-59-generic linux-modules-5.15.0-66-generic
[2023/09/20 05:24:48][INFO]   linux-modules-5.15.0-72-generic linux-modules-5.15.0-76-generic
[2023/09/20 05:24:48][INFO]   linux-modules-extra-5.13.0-35-generic linux-modules-extra-5.13.0-37-generic
[2023/09/20 05:24:48][INFO]   linux-modules-extra-5.15.0-43-generic linux-modules-extra-5.15.0-53-generic
[2023/09/20 05:24:48][INFO]   linux-modules-extra-5.15.0-54-generic linux-modules-extra-5.15.0-57-generic
[2023/09/20 05:24:48][INFO]   linux-modules-extra-5.15.0-59-generic linux-modules-extra-5.15.0-66-generic
[2023/09/20 05:24:48][INFO]   linux-modules-extra-5.15.0-72-generic linux-modules-extra-5.15.0-76-generic
[2023/09/20 05:24:48][INFO]   python2 python2-minimal python2.7 python2.7-minimal python3-systemd
[2023/09/20 05:24:48][INFO] Use 'sudo apt autoremove' to remove them.
[2023/09/20 05:24:48][INFO] The following additional packages will be installed:
[2023/09/20 05:24:48][INFO]   linux-hwe-5.15-tools-5.15.0-85
[2023/09/20 05:24:48][INFO] The following NEW packages will be installed:
[2023/09/20 05:24:48][INFO]   linux-hwe-5.15-tools-5.15.0-85 linux-tools-5.15.0-85-generic
[2023/09/20 05:24:48][INFO] 0 upgraded, 2 newly installed, 0 to remove and 1 not upgraded.
[2023/09/20 05:24:48][INFO] Need to get 6,352 kB of archives.
[2023/09/20 05:24:48][INFO] After this operation, 27.5 MB of additional disk space will be used.
[2023/09/20 05:24:48][INFO] Get:1 http://ports.ubuntu.com/ubuntu-ports focal-proposed/main arm64 linux-hwe-5.15-tools-5.15.0-85 arm64 5.15.0-85.95~20.04.2 [6,351 kB]
[2023/09/20 05:24:48][INFO] Get:2 http://ports.ubuntu.com/ubuntu-ports focal-proposed/main arm64 linux-tools-5.15.0-85-generic arm64 5.15.0-85.95~20.04.2 [1,948 B]
[2023/09/20 05:24:48][INFO] Fetched 6,352 kB in 2s (3,421 kB/s)
[2023/09/20 05:24:48][INFO] Selecting previously unselected package linux-hwe-5.15-tools-5.15.0-85.
[2023/09/20 05:24:48][INFO] (Reading database ...
[2023/09/20 05:24:48][INFO] (Reading database ... 5%
[2023/09/20 05:24:48][INFO] (Reading database ... 10%
[2023/09/20 05:24:48][INFO] (Reading database ... 15%
[2023/09/20 05:24:48][INFO] (Reading database ... 20%
[2023/09/20 05:24:48][INFO] (Reading database ... 25%
[2023/09/20 05:24:48][INFO] (Reading database ... 30%
[2023/09/20 05:24:48][INFO] (Reading database ... 35%
[2023/09/20 05:24:48][INFO] (Reading database ... 40%
[2023/09/20 05:24:48][INFO] (Reading database ... 45%
[2023/09/20 05:24:48][INFO] (Reading database ... 50%
[2023/09/20 05:24:48][INFO] (Reading database ... 55%
[2023/09/20 05:24:48][INFO] (Reading database ... 60%
[2023/09/20 05:24:48][INFO] (Reading database ... 65%
[2023/09/20 05:24:48][INFO] (Reading database ... 70%
[2023/09/20 05:24:48][INFO] (Reading database ... 75%
[2023/09/20 05:24:48][INFO] (Reading database ... 80%
[2023/09/20 05:24:48][INFO] (Reading database ... 85%
[2023/09/20 05:24:48][INFO] (Reading database ... 90%
[2023/09/20 05:24:48][INFO] (Reading database ... 95%
[2023/09/20 05:24:48][INFO] (Reading database ... 100%
[2023/09/20 05:24:48][INFO] (Reading database ... 594474 files and directories currently installed.)
[2023/09/20 05:24:48][INFO] Preparing to unpack .../linux-hwe-5.15-tools-5.15.0-85_5.15.0-85.95~20.04.2_arm64.deb ...
[2023/09/20 05:24:48][INFO] Unpacking linux-hwe-5.15-tools-5.15.0-85 (5.15.0-85.95~20.04.2) ...
[2023/09/20 05:24:48][INFO] Selecting previously unselected package linux-tools-5.15.0-85-generic.
[2023/09/20 05:24:48][INFO] Preparing to unpack .../linux-tools-5.15.0-85-generic_5.15.0-85.95~20.04.2_arm64.deb ...
[2023/09/20 05:24:48][INFO] Unpacking linux-tools-5.15.0-85-generic (5.15.0-85.95~20.04.2) ...
[2023/09/20 05:24:48][INFO] Setting up linux-hwe-5.15-tools-5.15.0-85 (5.15.0-85.95~20.04.2) ...
[2023/09/20 05:24:48][INFO] Setting up linux-tools-5.15.0-85-generic (5.15.0-85.95~20.04.2) ...
[2023/09/20 05:24:48][INFO] LTTng already installed.
[2023/09/20 05:24:48][INFO] 
[2023/09/20 05:24:48][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:48][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:48][INFO] debconf: unable to initialize frontend: Dialog
[2023/09/20 05:24:48][INFO] debconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)
[2023/09/20 05:24:48][INFO] debconf: falling back to frontend: Readline
[2023/09/20 05:24:48][INFO] debconf: unable to initialize frontend: Readline
[2023/09/20 05:24:48][INFO] debconf: (This frontend requires a controlling tty.)
[2023/09/20 05:24:48][INFO] debconf: falling back to frontend: Teletype
[2023/09/20 05:24:48][INFO] dpkg-preconfigure: unable to re-open stdin:
[2023/09/20 05:24:48][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:48][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:48][INFO] 
[2023/09/20 05:24:49][INFO] Invalid keyword LTTng_Kernel_ThreadKeyword, skipped...
[2023/09/20 05:24:49][INFO] Invalid keyword LTTng_Kernel_ContextSwitchKeyword, skipped...
[2023/09/20 05:24:49][INFO] Collection started.
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:49][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] =============== Iteration 0 ================
[2023/09/20 05:24:49][INFO] ***Test***
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EnableEventLog=1
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EventSourceFilter=PerfLabGenericEventSource
[2023/09/20 05:24:49][INFO] Added environment variable: DOTNET_STARTUP_HOOKS=/home/helixbot/work/B43C0999/p/PerfLabGenericEventSourceForwarder/PerfLabGenericEventSourceForwarder.dll
[2023/09/20 05:24:49][INFO] Hello World!
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] =============== Iteration 1 ================
[2023/09/20 05:24:49][INFO] ***Test***
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EnableEventLog=1
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EventSourceFilter=PerfLabGenericEventSource
[2023/09/20 05:24:49][INFO] Added environment variable: DOTNET_STARTUP_HOOKS=/home/helixbot/work/B43C0999/p/PerfLabGenericEventSourceForwarder/PerfLabGenericEventSourceForwarder.dll
[2023/09/20 05:24:49][INFO] Hello World!
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] =============== Iteration 2 ================
[2023/09/20 05:24:49][INFO] ***Test***
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EnableEventLog=1
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EventSourceFilter=PerfLabGenericEventSource
[2023/09/20 05:24:49][INFO] Added environment variable: DOTNET_STARTUP_HOOKS=/home/helixbot/work/B43C0999/p/PerfLabGenericEventSourceForwarder/PerfLabGenericEventSourceForwarder.dll
[2023/09/20 05:24:49][INFO] Hello World!
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] =============== Iteration 3 ================
[2023/09/20 05:24:49][INFO] ***Test***
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EnableEventLog=1
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EventSourceFilter=PerfLabGenericEventSource
[2023/09/20 05:24:49][INFO] Added environment variable: DOTNET_STARTUP_HOOKS=/home/helixbot/work/B43C0999/p/PerfLabGenericEventSourceForwarder/PerfLabGenericEventSourceForwarder.dll
[2023/09/20 05:24:49][INFO] Hello World!
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] =============== Iteration 4 ================
[2023/09/20 05:24:49][INFO] ***Test***
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EnableEventLog=1
[2023/09/20 05:24:49][INFO] Added environment variable: COMPlus_EventSourceFilter=PerfLabGenericEventSource
[2023/09/20 05:24:49][INFO] Added environment variable: DOTNET_STARTUP_HOOKS=/home/helixbot/work/B43C0999/p/PerfLabGenericEventSourceForwarder/PerfLabGenericEventSourceForwarder.dll
[2023/09/20 05:24:49][INFO] Hello World!
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:49][INFO] 
[2023/09/20 05:24:50][INFO] ...STOPPED.
[2023/09/20 05:24:50][INFO] Starting post-processing. This may take some time.
[2023/09/20 05:24:50][INFO] Compressing trace files
[2023/09/20 05:24:50][INFO] ...FINISHED
[2023/09/20 05:24:50][INFO] Cleaning up artifacts
[2023/09/20 05:24:50][INFO] ...FINISHED
[2023/09/20 05:24:50][INFO] Trace saved to Startup_-_Static_Console_Template_-_FDD_Publish_startup.trace.zip
[2023/09/20 05:24:50][INFO] 
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] tput: unknown terminal "unknown"
[2023/09/20 05:24:50][INFO] 
[2023/09/20 05:24:50][INFO] Parsing traces/Startup_-_Static_Console_Template_-_FDD_Publish_startup.trace.zip
[2023/09/20 05:24:51][INFO] parser = ScenarioMeasurement.TimeToMain2Parser
[2023/09/20 05:24:51][INFO] processName = staticconsoletemplate
[2023/09/20 05:24:51][INFO] pids = 407307, 407317, 407327, 407337, 407347
[2023/09/20 05:24:51][INFO] commandLine = "pub/staticconsoletemplate"
[2023/09/20 05:24:51][INFO] Copied traces/Startup_-_Static_Console_Template_-_FDD_Publish_startup.trace.zip to /home/helixbot/work/B43C0999/w/B77D09DF/uploads/Startup_-_Static_Console_Template_-_FDD_Publish_startup.trace.zip
[2023/09/20 05:24:51][INFO] Unhandled exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
[2023/09/20 05:24:51][INFO]  ---> System.ArgumentOutOfRangeException: Index was out of range. Must be non-negative and less than the size of the collection. (Parameter 'index')
[2023/09/20 05:24:51][INFO]    at System.Collections.Generic.List`1.get_Item(Int32 index)
[2023/09/20 05:24:51][INFO]    at Microsoft.Diagnostics.Tracing.Ctf.CtfReader.EnumerateEventHeaders()+MoveNext()
[2023/09/20 05:24:51][INFO]    at Microsoft.Diagnostics.Tracing.CtfTraceEventSource.ChannelEntry.MoveNext()
[2023/09/20 05:24:51][INFO]    at Microsoft.Diagnostics.Tracing.CtfTraceEventSource.ChannelListEnumerator.MoveNext()
[2023/09/20 05:24:51][INFO]    at Microsoft.Diagnostics.Tracing.CtfTraceEventSource.Process()
[2023/09/20 05:24:51][INFO]    at ScenarioMeasurement.TraceSourceManager.Process() in /home/vsts/work/1/s/src/tools/ScenarioMeasurement/Startup/TraceSourceManager.cs:line 39
[2023/09/20 05:24:51][INFO]    at ScenarioMeasurement.TimeToMain2Parser.Parse(String mergeTraceFile, String processName, IList`1 pids, String commandLine) in /home/vsts/work/1/s/src/tools/ScenarioMeasurement/Startup/TimeToMain2Parser.cs:line 99
[2023/09/20 05:24:51][INFO]    at ScenarioMeasurement.Startup.Main(String appExe, MetricType metricType, String scenarioName, String traceName, Boolean processWillExit, Int32 iterations, String iterationSetup, String setupArgs, String iterationCleanup, String cleanupArgs, Int32 timeout, Int32 measurementDelay, String appArgs, String logFileName, String workingDir, Boolean warmup, Boolean guiApp, Boolean skipProfileIteration, String reportJsonPath, String traceDirectory, String environmentVariables, String innerLoopCommand, String innerLoopCommandArgs, Boolean runWithoutExit, Int32 hotReloadIters, Boolean skipMeasurementIteration, Boolean parseOnly, Boolean runWithDotnet, Int32 affinity) in /home/vsts/work/1/s/src/tools/ScenarioMeasurement/Startup/Startup.cs:line 338
[2023/09/20 05:24:51][INFO]    at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
[2023/09/20 05:24:51][INFO]    at System.Reflection.MethodBaseInvoker.InvokeWithManyArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[2023/09/20 05:24:51][INFO]    --- End of inner exception stack trace ---
[2023/09/20 05:24:51][INFO]    at System.Reflection.MethodBaseInvoker.InvokeWithManyArgs(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
[2023/09/20 05:24:51][INFO]    at System.CommandLine.NamingConventionBinder.ModelBindingCommandHandler.InvokeAsync(InvocationContext context)
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Invocation.InvocationPipeline.<>c__DisplayClass4_0.<<BuildInvocationChain>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass17_0.<<UseParseErrorReporting>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass15_0.<<AddMiddleware>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass12_0.<<UseHelp>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass22_0.<<UseVersionOption>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass19_0.<<UseTypoCorrections>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<UseSuggestDirective>b__18_0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass16_0.<<UseParseDirective>b__0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c.<<RegisterWithDotnetSuggest>b__5_0>d.MoveNext()
[2023/09/20 05:24:51][INFO] --- End of stack trace from previous location ---
[2023/09/20 05:24:51][INFO]    at System.CommandLine.Builder.CommandLineBuilderExtensions.<>c__DisplayClass8_0.<<UseExceptionHandler>b__0>d.MoveNext()
[2023/09/20 05:24:51][ERROR] Process exited with status 1
[2023/09/20 05:24:51][INFO] Run failure registered
Traceback (most recent call last):
  File "test.py", line 13, in <module>
    Runner(traits).run()
  File "/home/helixbot/work/B43C0999/p/shared/runner.py", line 301, in run
    startup.runtests(self.traits)
  File "/home/helixbot/work/B43C0999/p/shared/startup.py", line 154, in runtests
    RunCommand(startup_args, verbose=True).run()
  File "/home/helixbot/work/B43C0999/p/scripts/performance/common.py", line 311, in run
    raise CalledProcessError(
subprocess.CalledProcessError: Command '$ /home/helixbot/work/B43C0999/p/startup/Startup --app-exe pub/staticconsoletemplate --metric-type TimeToMain2 --trace-name "Startup - Static Console Template - FDD Publish_startup" --gui-app false --process-will-exit true --iterations 5 --timeout 50 --warmup true --working-dir /home/helixbot/work/B43C0999/w/B77D09DF/e --report-json-path traces/perf-lab-report.json --trace-directory traces --scenario-name "Startup - Static Console Template - FDD Publish" --environment-variables COMPlus_EnableEventLog=1' returned non-zero exit status 1.
+ export _commandExitCode=1
```

_TraceEvent_ part tracked in https://github.com/microsoft/perfview/issues/1911.

--- a/src/tools/ScenarioMeasurement/MemoryConsumption/MemoryConsumption.csproj
+++ b/src/tools/ScenarioMeasurement/MemoryConsumption/MemoryConsumption.csproj
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.5" />
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.7" />
     <PackageReference Include="System.CommandLine.DragonFruit" Version="0.4.0-alpha.22272.1" />
     <PackageReference Include="System.Security.Principal.Windows" Version="4.7.0" />
   </ItemGroup>

--- a/src/tools/ScenarioMeasurement/PowerConsumption/PowerConsumption.csproj
+++ b/src/tools/ScenarioMeasurement/PowerConsumption/PowerConsumption.csproj
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.5" />
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.7" />
     <PackageReference Include="System.CommandLine.DragonFruit" Version="0.4.0-alpha.22272.1" />
     <PackageReference Include="System.Security.Principal.Windows" Version="4.7.0" />
   </ItemGroup>

--- a/src/tools/ScenarioMeasurement/Startup/Startup.csproj
+++ b/src/tools/ScenarioMeasurement/Startup/Startup.csproj
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.5" />
+    <PackageReference Include="Microsoft.Diagnostics.Tracing.TraceEvent" Version="3.1.7" />
     <PackageReference Include="System.CommandLine.DragonFruit" Version="0.4.0-alpha.22272.1" />
     <PackageReference Include="System.Security.Principal.Windows" Version="4.7.0" />
   </ItemGroup>

